modules/apps/object/object-api/src/main/java/com/liferay/object/util/ModelListenerUtil.java

```diff
@@ -0,0 +1,36 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+package com.liferay.object.util;
+
+import com.liferay.portal.kernel.audit.AuditMessage;
+import com.liferay.portal.security.audit.event.generators.util.Attribute;
+import com.liferay.portal.security.audit.event.generators.util.AuditMessageBuilder;
+
+import java.util.List;
+
+/**
+ * @author Marcela Cunha
+ */
+public class ModelListenerUtil {
+
+	public static AuditMessage getAuditMessage(
+		List<Attribute> attributes, String className, long classPK,
+		String eventType) {
+
+		return AuditMessageBuilder.buildAuditMessage(
+			eventType, className, classPK, attributes);
+	}
```

Comment:
@marcelabc I think Brian may remove this Util :/ because it is just a call to another method. Dont you think it's better to call the `AuditMessageBuilder.buildAuditMessage` directly?